src/annotator/integrations/html-side-by-side.ts

```diff
@@ -127,33 +116,29 @@ function elementContentRect(element) {
 /**
  * Yield all the text node descendants of `root` that intersect `rect`.
  *
- * @param {Element} root
- * @param {DOMRect} rect
- * @param {(el: Element) => boolean} shouldVisit - Optional filter that determines
- *   whether to visit a subtree
- * @return {Generator<Text>}
+ * @param shouldVisit - Optional filter that determines whether to visit a subtree
  */
-function* textNodesInRect(root, rect, shouldVisit = () => true) {
-  /** @type {Node|null} */
-  let node = root.firstChild;
+function* textNodesInRect(
+  root: Element,
+  rect: DOMRect,
+  shouldVisit: (el: Element) => boolean
```

Comment:
I have removed the default value, because this function is private to this module, and never called without a callback, which was making the coverage drop.